.gitignore

```diff
@@ -27,3 +27,6 @@ TrafficCapture/dockerSolution/src/main/docker/migrationConsole/staging
 
 # Directory used to temporarily store snapshots during testing
 DocumentsFromSnapshotMigration/docker/snapshots/
+transformation/standardJavascriptTransforms/coverage
```

Comment:
Nit; I prefer .gitignore files under the project, makes copying/refactoring easier in the future.